### PR TITLE
Gutenboarding: Fix welcome modal styling issues on Gutenberg 8.2.

### PR DESCRIPTION
**Pull request by yansern**
## Changes proposed in this Pull Request

This PR fixes the welcome modal styling issues on Gutenberg 8.2.

This PR also fixes the welcome modal styling on mobile view. Following closely to this design https://github.com/Automattic/wp-calypso/issues/42141#issuecomment-627881326 but without the dots at the bottom (due to alignment issues on tight spaces) which I temporarily just hide it for now til design is finalized. See https://github.com/Automattic/wp-calypso/issues/42141#issuecomment-638884014.

This PR also adds a bit of UX improvement by ensuring the footer position always stay consistent. This is so user does not have to reposition the cursor to click on the Next/Previous button when user goes to a different slide on the welcome modal.

## Testing instructions

Run `yarn dev --sync` on folder `apps/full-site-editing` and test on the block editor on the sandbox.

To ensure you always see the gutenboarding welcome modal on the block editor (without having to go through the launch flow:

Comment out these lines in `wpcom-nux.js`.
```
if ( ! isWpcomNuxEnabled || isSPTOpen ) {
    return null;
}
```

And also locate and replace the value of `isGutenboarding` in `wpcom-nux.js`.
```
const isGutenboarding = true;
```

## Screenshot

The gif below simulates what happens (by reducing modal height) when text from different language takes up more content height. It will just expands the content height upwards while keeping the footer position consistent, so user does not have to move the cursor again to click on the next/previous button.

Here's the mobile view (without the dots for now).

Fixes https://github.com/Automattic/wp-calypso/issues/42946

### CODE REVIEW

File: apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nux/src/style.scss [hunk 1]
```diff
@@ -30,17 +52,74 @@ $wpcom-modal-padding-h: 50px;
...
+		width: 100%;
+		height: $wpcom-modal-footer-height;
+		bottom: $wpcom-modal-footer-height * -1;
+		left: 0;
+		padding: $wpcom-modal-footer-padding-v 0;
+		margin: 0;
+		display: flex;
+		justify-content: center;
+		background: white;
+		border-top: 1px solid #dcdcde;
```
Comment (simison) [inline apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nux/src/style.scss:66]: Any Gutenberg sass variable we could use here? From `@import '~@wordpress/base-styles/colors';`

Comment (simison) [reply apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nux/src/style.scss:66]: sounds good 👍

**Comment by simison** [comment]
Looks great! I can't dismiss it at the end tho with "get started" button — it just stays.

image

Testing with Chrome.

**Comment by yansern** [comment]
> Looks great! I can't dismiss it at the end tho with "get started" button — it just stays.
> 
> image
> 
> Testing with Chrome.

It's not dismissable if this hack is applied.
```
if ( ! isWpcomNuxEnabled || isSPTOpen ) {
    return null;
}
```

Remove that and it'll work.

**Comment by simison** [comment]
Yep! You're right. 

image

Also tested the welcome modal that opens from this menu.

**Pull request merged**